More elaborated html file

Ex-tela opened the pull request:
More elaborated template regarding bootstrap template (base) to insert new objects on the page.

Ex-tela committed:
More elaborated html file

More elaborated template regarding bootstrap template (base) to insert new objects on the page.

mvallebr commented on hello_world.html:
```diff
@@ -6,12 +6,44 @@
-    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/js/bootstrap.min.js" integrity="sha384-ChfqqxuZUCnJSK3+MXmPNIyE6ZbWh2IMqE241rYiqJxyMiZ6OW/JmZQ5stwEULTy" crossorigin="anonymous"></script>
+    <import stylesheet from"../node_modules/bootstrap/scss/bootstrap"></import>
```
what are you doing in here? you're importing from a relative path `../` - you shouldn't import anything from local, import everything from bootstrap cdn. 
you would use this relative path if you had created a file yourself to be imported, which is not the case here

mvallebr commented on hello_world.html:
```diff
@@ -6,12 +6,44 @@
+      </div>
+    </header>
+    <div class="h-50 d-inline-block p-" style="width: 900px; background-color: rgba(255, 255, 255, 0.1)">
+      <input type="text" class="form-control" placeholder="Insert here Chords" aria-label="Insert here Chords" aria-describedby="basic-addon2">
```
you probably want a bigger editor...

mvallebr commented on hello_world.html:
```diff
@@ -6,12 +6,44 @@
   <body>
-    <h1>Hello, world!</h1>
+    <header class="p-3 bg-primary text-white">
+      <div class="container">
```
wouldn't container fluid be better?

Ex-tela committed:
Diary update - draft pull request

Ex-tela committed:
Add files via upload

Ex-tela committed:
To push the current branch and set the remote as upstream, use
Please enter the commit message for your changes. Lines starting

This is so confusing, but if it works it's an update

	

	This is so confusing, but if it works it's an update

Ex-tela committed:
new file added (chord_gen_rest_py)

Ex-tela committed:
conflicts solved
